week_5/char-yb/week5/src/main/kotlin/com/sipe/week5/global/util/security/SecurityUtil.kt

```
@@ -0,0 +1,20 @@
+import org.springframework.security.core.Authentication
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.stereotype.Component
+
+@Component
+class SecurityUtil {
+	val currentMemberId: Long
+		get() {
+			val authentication: Authentication = SecurityContextHolder.getContext().authentication
+			try {
+				return authentication.name.toLong()
+			} catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)